### ISSUE TITLE
[plot] Add support for image rotation

### ISSUE THREAD

**Issue by t20100** [silx-kit/silx#1946]
Add support for more transformation of the image in the plot

**Comment by vallsv** [comment]
Be careful that you also have to maintain the mask or things like that.

**Comment by t20100** [comment]
Yes, it is already the case for offset and scale... It will probably need a better unified handling of transformation.
